### PR TITLE
chore(makefile): try cypress image

### LINKED ISSUE #43103: E2E Tests: Run cypress from docker image

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What this PR does / why we need it**:
- Trying to run cypress in a docker image provided by cypress

**Which issue(s) this PR fixes**:

Fixes: https://github.com/grafana/grafana/issues/43103

Screenshot 2021-12-24 at 12 56 46

**Special notes for your reviewer**:

### CODE REVIEW

File: scripts/drone/pipelines/pr.star [hunk 1]
```diff
@@ -93,7 +93,7 @@ def pr_pipelines(edition):
     # Insert remaining build_steps
     build_steps.extend([
         package_step(edition=edition, ver_mode=ver_mode, include_enterprise2=include_enterprise2, variants=variants),
-        install_cypress_step(),
+        #install_cypress_step(),
```
Comment (dsotirakis) [inline scripts/drone/pipelines/pr.star:96]: This can be removed.

**Review by dsotirakis — APPROVED** [review]

File: scripts/drone/pipelines/pr.star (hunk 1 above)
Comment (tolzhabayev) [reply scripts/drone/pipelines/pr.star]: @dsotirakis done

**Review by dsotirakis — APPROVED** [review]

**Comment by tolzhabayev** [comment]
@dsotirakis I will not merge it now, will do it next week. It failed a couple of times with a weird error message about `mkdir`, I will run it a couple of times more to see how stable it is.

**Comment by tolzhabayev** [comment]
@dsotirakis ok so I let it run a couple of times and did not bump into any issues, all works pretty stable. I will merge it now.

**Pull request merged**